pkgs/development/tools/xcbuild/sdk.nix

```diff
@@ -12,7 +12,7 @@ let
   };
 
   SystemVersion = {
-    ProductName = "Mac OS X";
```

Comment:
Version 10.10 is called "OS X", so i corrected it.
I don't know if we could update `ProductVersion` to 10.13 (newest version) and use "macOS" here.

Thanks for the review!